### PR TITLE
Propagate Consumers status to ConsumerGroup 

### PR DESCRIPTION
**Pull request by pierDipi**
Propagate subscriberURI, deadLetterSinkURI,
and ready replica count.

Part of #1537

### CODE REVIEW

File: control-plane/pkg/apis/internals/kafka/eventing/v1alpha1/consumer_group_types.go [hunk 1]
```diff
@@ -191,3 +191,13 @@ func (cg *ConsumerGroup) IsNotScheduled() bool {
 	cond := cg.Status.GetCondition(ConditionConsumerGroupConsumersScheduled)
 	return cond.IsFalse() || cond.IsUnknown()
 }
+
+func (in *ConsumerGroup) HasDeadLetterSink() bool {
+	return hasDeadLetterSink(in.Spec.Template.Spec.Delivery)
+}
```
Comment (pierDipi) [inline control-plane/pkg/apis/internals/kafka/eventing/v1alpha1/consumer_group_types.go:197]:
```suggestion
func (cg *ConsumerGroup) HasDeadLetterSink() bool {
	return hasDeadLetterSink(cg.Spec.Template.Spec.Delivery)
}
```

File: control-plane/pkg/apis/internals/kafka/eventing/v1alpha1/consumer_group_types.go [hunk 2]
```diff
@@ -191,3 +191,13 @@ func (cg *ConsumerGroup) IsNotScheduled() bool {
 	cond := cg.Status.GetCondition(ConditionConsumerGroupConsumersScheduled)
 	return cond.IsFalse() || cond.IsUnknown()
 }
+
+func (in *ConsumerGroup) HasDeadLetterSink() bool {
```
[the remark this answers is not in the document]
Comment (pierDipi) [reply control-plane/pkg/apis/internals/kafka/eventing/v1alpha1/consumer_group_types.go]: Done

**Review by aavarghese — APPROVED** [review]
/lgtm

**Pull request merged**